studio/components/interfaces/Organization/BillingSettings/TaxID/TaxID.tsx

```diff
@@ -132,12 +133,33 @@ const TaxID: FC<Props> = ({ loading, taxIds, onTaxIdsUpdated }) => {
     }
   }
 
+  /**
+   * Sanitize EU VAT ids so they get prepended with the country code
+   *
+   * Ex: Belgium's VAT ID should be BE0123456789 not 0123456789
+   * Only EU VAT ids are affected for now
+   */
+  const sanitizeTaxID = (taxId: any) => {
```

Comment:
shall we get this typed properly ?

Reply:
hmm can we shift this out into BillingSettings/TaxID.utils.ts

and then write some tests for it? under tests/components/Billing/TaxID.utils.test.js - give it a couple of test cases using eu tax ids and non eu tax ids